src/Microsoft/Silk.NET.Direct2D/WindowsCodecs/WindowsCodecs.gen.cs

```diff
@@ -0,0 +1,6011 @@
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(Guid* guidMetadataFormat, [UnmanagedType(Silk.NET.Core.Native.UnmanagedType.LPWStr)] string pwzSchema, uint cchName, [UnmanagedType(Silk.NET.Core.Native.UnmanagedType.LPUTF8Str)] string wzName, ref uint pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, char* wzName, uint* pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, char* wzName, ref uint pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, ref char wzName, uint* pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, ref char wzName, ref uint pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, [UnmanagedType(Silk.NET.Core.Native.UnmanagedType.LPUTF8Str)] string wzName, uint* pcchActual);
+
+        /// <summary>To be documented.</summary>
+        [NativeName("Src", "Line 7454, Column 16 in wincodec.h")]
+        [NativeApi(EntryPoint = "WICMapSchemaToName", Convention = CallingConvention.StdCall)]
+        public unsafe partial int WICMapSchemaToName(ref Guid guidMetadataFormat, char* pwzSchema, uint cchName, [UnmanagedType(Silk.NET.Core.Native.UnmanagedType.LPUTF8Str)] string wzName, ref uint pcchActual);
```

Comment:
It's odd that these still have the prefix, I think if you omit the underscore from the "prefix" configuration property it should fix this (the underscore is automatically added where appropriate)